addons/l10n_it/i18n/it.po

```diff
@@ -0,0 +1,1935 @@
+# Translation of Odoo Server.
```

Comment:
The i18n folder of l10n_it should be removed.  It is useless diff and the preceding subtotal can be done in l10n_it_edi_withholding.